Quantum-Control-Applications/Superconducting/Single Fixed Transmon/configuration.py

```diff
@@ -80,6 +80,9 @@ def readout_macro(threshold=None, state=None, I=None, Q=None):
 square_pi_len = 100
 square_pi_amp = 0.1
 
+drag_coef = 0
+drag_detun = 0
```

Comment:
This should be assign to delta in the DRAG functions, right?

Reply:
I will change the name to drag_detun to anaharmoncity. Delta = 1/anaharmonicity

drag_coef = alpha in our pyqua tools

Reply:
I put in the waveform already the paramter drag_coef to alpha in our waveforms

Reply:
Ok cool, then I would also write delta=1/anharmonicity in the function so that it is actually used

Reply:
yes, done